doc/coq.rst

```diff
@@ -98,6 +99,12 @@ The semantics of the fields are:
   is taken from the value set in the ``(coq (flags <flags>))`` field in ``env``
   profile. See :ref:`dune-env` for more information.
 
+- ``<stdlib_included>`` can either be ``yes`` or ``no``, currently defaulting to
+  ``yes``. When set to ``no``, the theory will be built without binding Coq's
+  standard library to the `Coq` module and without importing ``Coq.Init.Prelude``.
+  To achieve this, in addition to ``<coq_flags>``, the extra flags ``-boot -noinit``
+  will be passed to ``coqc``.
```

Comment:
and coqdep right?

Reply:
Yeah, I thought that this was information that is kind of internal and does not need to be mentioned in the manual. Do you disagree?

Reply:
OK let's keep it as is then.